playwright-tests/tests/page-objects/theia-trace-viewer-app.ts

```diff
@@ -0,0 +1,39 @@
+/********************************************************************************
+ * Copyright (C) 2022 EclipseSource and others.
```

Comment:
Shouldn't be the copyright Ericsson and others? I think you wrote this file.

 2024?

Reply:
In the latest version, I have kept the original copyright header as-is (like above) and added a second header that has "Ericsson and others" as owner and the current year.